package.json

```diff
@@ -4,8 +4,10 @@
   "homepage": "https://risocos.github.io/regalos",
   "private": true,
   "dependencies": {
+    "moment": "^2.22.1",
     "npm": "^6.0.0",
     "react": "^16.3.2",
+    "react-datepicker": "^1.4.1",
```

Comment:
Als `react-datepicker` `moment` nodig heeft, dat moet die hem toch als dependency hebben? Waarom add jij beide. De package.json van react-datepicker zou dan moment moeten includen als die hem echt nodig heeft. Of is dit echt niet anders te doen?

Reply:
Ik moest hem er bij doen, ik probeerde zonder maar is een dependency die je nog los moet installeren blijkbaar